context/app/static/js/shared-styles/alerts/InfoAlert.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import Stack from '@mui/material/Stack';
+import Paper from '@mui/material/Paper';
+import Typography from '@mui/material/Typography';
+
+import { InfoIcon } from 'js/shared-styles/icons';
+
+function InfoAlert({ children }: { children: React.ReactNode }) {
+  return (
+    <Stack component={Paper} p={2} spacing={2} marginBottom={1.25}>
```

Comment:
1.25 margin evaluates to 10px, I believe. This isn't a value I see often in the design. Did this come from the design?

Reply:
The designs actually specify that this should be 8px, so I'll update to `marginBottom={1}`.